src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let total = days * 40;
+
+  if (days >= 7) {
+    total -= 50;
```

Comment:
Could I suggest making the discounts into a variable, like `const soManyDaysDiscount = 100` for example. That way, if the discount amount changes in the future, it can easily be changed.